client/app/dedicated/server/translations/Messages_fr_FR.json

```diff
@@ -347,6 +347,9 @@
   "server_configuration_datacenter": "Datacentre",
   "server_configuration_rack": "Baie",
   "server_configuration_server_id": "Serveur ID",
+  "server_configuration_upgrade": "Upgrade",
```

Comment:
`Mise à niveau` ?

Reply:
Should be better, but this translation comes from the mock up.

Reply:
CX doesn't decide the texts ;)

Reply:
Updated

Reply:
@FredericEspiau @AxelPeter FYI, often CX includes Content team work in the mock ups